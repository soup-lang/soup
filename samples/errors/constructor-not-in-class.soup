func x()
    ctor(m, n)
        this.m: m
        this.n: n
