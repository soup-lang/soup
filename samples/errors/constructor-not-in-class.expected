samples/errors/constructor-not-in-class.soup:2
    constructors not allowed outside class definition
